README.md

```diff
@@ -85,22 +85,20 @@ The following variants support ECS:
 
 We also have variants that are designed to be Kubernetes worker nodes in VMware:
 
-* `vmware-k8s-1.26`
 * `vmware-k8s-1.27`
 * `vmware-k8s-1.28`
 * `vmware-k8s-1.29`
 
 The following variants are designed to be Kubernetes worker nodes on bare metal:
 
-* `metal-k8s-1.26`
 * `metal-k8s-1.27`
 * `metal-k8s-1.28`
 * `metal-k8s-1.29`
 
 The following variants are no longer supported:
 
 * All Kubernetes variants using Kubernetes 1.22 and earlier
-* Bare metal and VMware variants using Kubernetes 1.25 and earlier
+* Bare metal and VMware variants using Kubernetes 1.26 and earlier
```

Comment:
This is fine but it seems like we missed adding `vmware-k8s-1.30` to the list above

Reply:
thanks for the catch - I added a commit to this PR to include missing k8s 1.30 variants docs